indico/modules/events/abstracts/templates/abstract/review.html

```diff
@@ -150,204 +75,66 @@
-                {% elif review.proposed_action.name == 'change_tracks' %}
-                    <div class="">
-                        {% set tracks -%}
-                            {% for track in review.proposed_tracks -%}
-                                <span class="abstract-track">{{ track.title|escape }}</span>
-                                {%- if not loop.last -%}, {% endif -%}
-                            {% endfor %}
-                        {%- endset %}
-                        {% trans count=review.proposed_tracks|length, tracks=tracks|safe -%}
-                            Possible destination track: {{ tracks }}
-                        {%- pluralize -%}
-                            Possible destination tracks: {{ tracks }}
-                        {%- endtrans %}
-                    </div>
-                {% elif review.proposed_action.name in ('mark_as_duplicate', 'merge') %}
-                    <div class="">
-                        {% set other_abstract = review.proposed_related_abstract %}
-                        {% set abstract_link -%}
-                            <a href="{{ url_for('.display_abstract', other_abstract) }}">
-                                {{ other_abstract.title|escape }}
-                            </a>
-                        {%- endset %}
-                        {% if review.proposed_action.name == 'merge' %}
-                            {% trans title=abstract_link|safe -%}
-                                Proposed to be merged into {{ title }}
-                            {%- endtrans %}
-                        {% elif review.proposed_action.name == 'mark_as_duplicate' %}
-                            {% trans title=abstract_link|safe -%}
-                                Proposed as duplicate of {{ title }}
-                            {%- endtrans %}
-                        {% endif %}
-                    </div>
-                {% endif %}
-                {% if review.ratings %}
-                    <div class="titled-rule">
-                        {% trans %}Ratings{% endtrans %}
-                    </div>
-                    {% for rating in review.ratings|sort(attribute='question.position') %}
-                        <ul class="abstract-question-list">
+                </button>
+                <button class="i-button big text-color color-on-hover danger {{ 'js-new-cancel' if not comment }}"
+                        data-button-back>
+                    {% trans %}Cancel{% endtrans %}
+                </button>
+            {% endcall %}
+        </div>
+        {% if abstract.can_review(session.user, check_state=true) %}
+            {% set tracks = abstract.get_reviewed_for_tracks_by_user(session.user) %}
+            <div class="review-trigger flex">
+                <span class="comment-or-review">
+                    {%- trans %}or{% endtrans -%}
+                </span>
+                {% if tracks|length == 1 %}
+                    {% set tracks = tracks|list %}
```

Comment:
` {% set track = tracks|first %}` (actually since you only need it once you could even use `tracks|first` below without assigning it first)